docs/src/pages/customization/styling-engine/styling-engine.md

```diff
@@ -0,0 +1,27 @@
+# Styling engine
+
+<p class="description">Configure your preferred styling engine.</p>
+
+The engine used for generating the styles for the Material-UI components is [`@emotion`](https://github.com/emotion-js/emotion). Although this is the default configuration, there is a possibility for using [`styled-components`](https://github.com/styled-components/styled-components). In other to support this, there are two packages exported:
```

Comment:
One sentence per line helps reviewability.

> In other to support this, there are two packages exported

What is this supposed to mean?

Reply:
Updated.